### PR TITLE
Lots of UI updates

### PR DESCRIPTION
**Pull request by dwaxemberg**
- Added support for displaying multiple grids in the UI
- Added a toggle for train vs test data sets. 
- Added a new resource file for different line transformation.
- Cleaned up some of the UI code.
  - 100% BoxLayout now. 
  - Used `apply` to make consecutive adds cleaner.

### CODE REVIEW

File: src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt [hunk 1]
```diff
@@ -1,56 +1,143 @@
...
+import javax.swing.JLabel
+import javax.swing.JPanel
+import javax.swing.JToggleButton
 
-class ARCWindow(width : Int, height : Int, title : String, task : Task) : JFrame() {
+private const val GAP_SIZE = 12
+private const val GRID_SIZE = 40
 
-    private lateinit var gridPane : GridPane
+typealias TaskGrids = Pair<List<GridPane>, List<GridPane>>
```
Comment (deansher) [inline src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt:19]: This doesn't fit the logical structure of an ARC task, which has pairs of (input grid, output grid). The logical structure becomes important visually when grid sizes vary. Here's an [example](https://snipboard.io/D7c340.jpg).

File: src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt [hunk 2]
```diff
@@ -1,56 +1,143 @@
...
+                    isSelected = true
+                }
+            }
+        }
+    }
+
+    private fun createFromTask(task: Task): TaskGrids {
+        fun gridPane(serializedGrid: SerializedGrid): GridPane {
+            return GridPane(Grid(serializedGrid)).apply {
+                preferredSize = Dimension(GRID_SIZE, GRID_SIZE)
```
Comment (deansher) [inline src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt:124]: `GridPane`s  should not be as "stretchy" as these end up being. I'm not quite sure what the rules should be, but see for example [this PR's rendering](https://snipboard.io/uRFEJb.jpg) of task e98196ab versus the official [ARC testing interface](https://snipboard.io/D7c340.jpg).

I think it's fine to improve this either in this PR or in a subsequent PR.

**Review by deansher — CHANGES_REQUESTED** [review]

File: src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt (hunk 2 above)
Comment (dwaxemberg) [reply src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt:124]: I'll address this in #13

**Comment by dwaxemberg** [comment]
merged acd14c6

**Pull request closed without merging**